bin/http-server

```diff
@@ -18,41 +56,40 @@ if (argv.h || argv.help) {
     'usage: http-server [path] [options]',
     '',
     'options:',
-    '  -p           Port to use [8080]',
-    '  -a           Address to use [0.0.0.0]',
-    '  -d           Show directory listings [true]',
-    '  -i           Display autoIndex [true]',
-    '  -g --gzip    Serve gzip files when possible [false]',
-    '  -e --ext     Default file extension if none supplied [none]',
-    '  -s --silent  Suppress log messages from output',
-    '  --cors[=headers]   Enable CORS via the "Access-Control-Allow-Origin" header',
-    '                     Optionally provide CORS headers list separated by commas',
-    '  -o [path]    Open browser window after starting the server',
-    '  -c           Cache time (max-age) in seconds [3600], e.g. -c10 for 10 seconds.',
-    '               To disable caching, use -c-1.',
-    '  -U --utc     Use UTC time format in log messages.',
+    '  -p\t           Port to use [Default: 8080]',
+    '  -a\t           Address to use [Default: 0.0.0.0]',
+    '  -d\t           Show directory listings. Use --no-d to disable [Default: enabled]',
+    '  -i\t           Display autoIndex. Use --no-i to disable [Default: enabled]',
+    '  -g --gzip\t   Serve gzip files when possible [Default: disabled]',
+    '  -e --ext\t   Default file extension if none supplied [Default: "html"]',
+    '  -s --silent\t   Suppress log messages from output [Default: disabled]',
+    '  --cors[=headers] Enable CORS via the "Access-Control-Allow-Origin" header [Default: disabled]',
+    '                   Optionally provide CORS headers list separated by commas',
+    '  -o [path]\t   Open browser window after starting the server [Default: disabled]',
+    '  -c\t           Cache time (max-age) in seconds [Default: 3600], e.g. -c10 for 10 seconds.',
+    '    \t           To disable caching, use -c-1.',
+    '  -U --utc\t   Use UTC time format in log messages. [Default: disabled]',
     '',
-    '  -P --proxy   Fallback proxy if the request cannot be resolved. e.g.: http://someurl.com',
+    '  -P --proxy\t   Fallback proxy if the request cannot be resolved. e.g.: http://someurl.com',
     '',
-    '  -S --ssl     Enable https.',
-    '  -C --cert    Path to ssl cert file (default: cert.pem).',
-    '  -K --key     Path to ssl key file (default: key.pem).',
+    '  -S --ssl\t   Enable https [Default: disabled]',
+    '  -C --cert\t   Path to ssl cert file [Default: cert.pem]',
+    '  -K --key\t   Path to ssl key file [Default: key.pem]',
```

Comment:
Thanks for making these match the others!

Reply:
👍 Makes it more clear, I think